mara_app/static/mara.css

```diff
@@ -284,27 +284,28 @@ body > #alerts {
     text-overflow: ellipsis;
 }
 
-#mara-main div.card div.card-block > div.card-block-content {
+#mara-main div.card div.card-body > div.card-text,
+#mara-main div.card div.list-group > div.list-group-item {
     overflow-x: auto;
     overflow-y: hidden;
 }
 
-#mara-main div.card div.card-block > div.card-block-content > *:first-child {
+#mara-main div.card div.list-group > div.list-group-item > *:first-child {
```

Comment:
Huih, this might need an extra changelog entry that one should check the UI of the final mara app, especially if one is using some bootstrap styling in custom modules...

Reply:
Yea, right... this breaking change was introduced by 4.0. Mara still runs on a alpha version of bootstrap (#33)

Reply:
This change requires https://github.com/mara/mara-pipelines/pull/66 to be merged as well. Do you have an idea how to tell `pip` that the next mara-app version 2.4 requires the next mara-pipelines version 3.4.0?

Currently there  is no requirement between mara-app and mara-pipelines. Is there a way to tell that when package XYZ is installed, it must be above or below version 2.4/3.4.0?

Reply:
I do not know about any way to exclude certain packages, just depend on certain versions.

If there is a requirement, I would say add one? Or why do they need to be installed together?

Reply:
It looks like that default targets are not supported. See [here](https://stackoverflow.com/questions/60842775/how-to-package-a-python-module-with-extras-as-default). I think I will just merge it in the next minor version and make a note in the changelog in both repositories.